togetherly/lib/providers/chore_provider.dart

```diff
@@ -25,19 +26,42 @@ class ChoreProvider with ChangeNotifier {
 
   List<Chore> _allChores = [];
   List<Chore> get allChores => _allChores;
+  List<Chore> get choreListDueToday => _allChores
+      .where((chore) => chore.dueDate == DateHelpers.getDateToday())
+      .toList();
+  List<Chore> get choreListComingSoon =>
+      _allChores.where((chore) => _isChoreDueTomorrow(chore.dueDate)).toList();
+  List<Chore> get choreListOverdue =>
```

Comment:
I think it would be more consistent to name these three getters `choresDueToday`, `choresDueSoon`, and `overdueChores`/`choresOverdue`, rather than to include `list` in each of their names (especially as these will need to be changed to iterables rather than lists).

Reply:
Done, and changed to iterables